.github/dependabot.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    target-branch: "main"
+    allow:
+      - dependency-type: direct
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "GitHub Actions"
+      include: "scope"
+    labels:
+      - "dependencies"
+      - "workflows"
+
+  # Maintain dependencies for npm
+  - package-ecosystem: "npm"
+    directory: "/"
+    target-branch: "main"
+    allow:
+      - dependency-type: direct
+    schedule:
+      interval: "daily"
```

Comment:
Should we change this to weekly? In my experience, weekly + grouped is less noisy and a bit easier.